src/index.ts

```diff
@@ -72,6 +71,9 @@ class VideoSnapshot {
         video.pause();
         resolve(video);
       });
+      video.addEventListener('error', () => {
+        reject('failed to load video');
```

Comment:
will there be an argument for that callback? (error itself)
Also, it is not cool to throw strings as an error. how about we create an error object, maybe attach original one somehow if we get anything here as an argument?

Reply:
the argument is the event, which contains no useful information about the error. I will create an Error and throw it, that's a good idea